app-api/src/main/java/org/openmrs/eip/app/CustomFileOffsetBackingStore.java

```diff
@@ -1,86 +0,0 @@
-package org.openmrs.eip.app;
```

Comment:
Let's rename this file before we merge the work otherwise I can't see the exact diff

Reply:
I first merged the code and then renamed as you requested, to maintain the history, as you can see in the commit below:
5c6dd58947d717e0cf67a3cea7d87c4e6f92d07b
`rename from app-api/src/main/java/org/openmrs/eip/app/CustomFileOffsetBackingStore.java
rename to app-api/src/main/java/org/openmrs/eip/app/CustomDatabaseOffsetBackingStore.java
index 85e596b..207f1be 100644`